example-plugins/example_patient_sync/protocols/patient_sync.py

```diff
@@ -0,0 +1,155 @@
+from typing import Any
+
+from canvas_sdk.effects import Effect
+from canvas_sdk.effects.patient import CreatePatientExternalIdentifier
+from canvas_sdk.events import EventType
+from canvas_sdk.protocols import BaseProtocol
+from canvas_sdk.utils import Http
+from canvas_sdk.v1.data.patient import Patient
+from logger import log
+
+PARTNER_URL_BASE = "https://app.usebridge.xyz"
+SYSTEM_URL = "https://canvas.app.usebridge.xyz"
```

Comment:
Remove vendor-specific URLs. I think the code is actually using a mixture of these values and the ones set in the secrets.

Reply:
ahhh yes thank you for the reminder